src/components/Dashboard/index.js

```diff
@@ -81,14 +85,18 @@ class Dashboard extends React.Component {
 
           <div className="m-dashboard-panel">
             <div className="dashboard-header">
-              <button
-                className="text text-cta btn-filters-switcher"
-                onClick={ this.props.toggleFiltersFn } >
-                filters
-              </button>
-              <a href="http://www.care.org/donate" rel="noreferrer" target="_blank"className="btn btn-contrast -small">
-                Donate
-              </a>
+              { !this.props.embed &&
+                <button
+                  className="text text-cta btn-filters-switcher"
+                  onClick={ this.props.toggleFiltersFn } >
+                  filters
+                </button> }
+
+              { !this.props.embed ?
+                <a href="http://www.care.org/donate" rel="noreferrer" target="_blank" className={ `btn btn-contrast -small` }>
+                  Donate
+                </a> :
+                <a href="#" className="btn btn-primary btn-embed">Explore the map</a> }
             </div>
             <div className="scroll-wrapper">
               <DashDates
```

Comment:
Having a look at the design, I notice that there are some more styles missing. 
For example, as we are no longer having timeline, dashboard should be positioned to the very bottom of the screen and buttons to switch between projects and donations should take all the available with.